src/Cake.Common/Tools/VSWhere/Latest/VSWhereLatestSettings.cs

```diff
@@ -9,5 +9,9 @@ namespace Cake.Common.Tools.VSWhere.Latest
     /// </summary>
     public sealed class VSWhereLatestSettings : VSWhereSettings
     {
+        /// <summary>
+        /// Gets or sets the products to find. Defaults to Community, Professional, and Enterprise. Specify "*" by itself to search all product instances installed.
+        /// </summary>
+        internal string Products { get; set; }
```

Comment:
This needs so be public to be accessible from Cake scripts.

Reply:
I changed it to public.